velox/docs/functions/spark/datetime.rst

```diff
@@ -70,6 +70,11 @@ These functions support TIMESTAMP and DATE input types.
 
     This is an alias for :func:`day_of_week`.
 
+.. function:: get_timestamp(string, dateFormat) -> timestamp
+
+    Returns timestamp by parsing ``string`` according to the specified ``dateFormat``.
+    Returns null for parsing error.
```

Comment:
Can we clarify the behavior when `dateFormat` is null? Please also add some valid and invalid examples.
https://github.com/apache/spark/blob/4f65413031d8a196a3442fd39cedd50eaf1a7655/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala#L1271

Reply:
Just documented. Thanks!